tools/pot/openvino/tools/pot/algorithms/quantization/fake_quantize_configuration.py

```diff
@@ -328,7 +328,8 @@ def _is_concat_unify_condition(node):
         def _is_followed_by_conv(input_node):
             if _is_quantize_agnostic_op(input_node):
                 concat_stack.extend(get_all_node_outputs(input_node))
-            elif input_node.type in [n['type'] for n in CONCAT_UNIFY_OUTPUTS]:
+            elif input_node.type in [n['type'] for n in CONCAT_UNIFY_OUTPUTS] \
+                and not is_depth_wise(input_node):
```

Comment:
Do you think that GNA is able to execute quantized DW Conv in a per-channel fashion? I think it should be verified.

Reply:
Not sure, thanks for the comment. It was a general fix, that causes failure for ANY device quantization.
Do you think that DW Conv's exists at the GNA-like networks?

Reply:
@nkogteva, can you please comment here? I remember you had some concerns about DW Conv quantization.

Reply:
@AlexKoff88 We have a transformation that converts a DW separable convolution to a set of MatMul + Add. And we agreed at the meeting that this transformation should be moved to offline (POT). I created ticket 69672 for that.

Reply:
Ok. But let's handle this separately. Is this really a set of MatMul +Add?